mathics/format/asy.py

```diff
@@ -750,4 +750,21 @@ def uniform_polyhedron_3d_box(self: RectangleBox, **options) -> str:
     )
 
 
+# def tetrahedron_polyhedron_3d_box(self: RectangleBox, **options) -> str:
```

Comment:
How is this related to the other changes?

Reply:
It isn't. I had been working on that and this was on the disk when I committed. Rather than create another branch, may as well commit what I had so that later I can come back to this (after release probably).

Reply:
Ok - I now have all this code (and other code that was on my local disk) put in a local branch. So this is now removed.

After CI clears I will merge this. There a still a bit more on the miscellaneous list functions.